From bd95fea1eaeaa8646203d2934dc51c1a1cbf72b6 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 26 Feb 2021 16:18:21 +0800
Subject: [PATCH] scripts/gcc-wrapper.py: ignore vfs.c:1259

fs/incfs/vfs.c: In function 'ioctl_create_file':
fs/incfs/vfs.c:1259:10: warning: cast to pointer from integer of different size [-Wint-to-pointer-cast]

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I4a9e167927d32180871776767fe3e4ff0c8342a9
---
 scripts/gcc-wrapper.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/gcc-wrapper.py b/scripts/gcc-wrapper.py
index 57bc40f6aa02..f8bc043a66a1 100755
--- a/scripts/gcc-wrapper.py
+++ b/scripts/gcc-wrapper.py
@@ -38,6 +38,7 @@ import sys
 import subprocess
 
 allowed_warnings = set([
+    "vfs.c:1259", # fs/incfs/vfs.c:1259:10: warning: cast to pointer from integer of different size [-Wint-to-pointer-cast]
     "pseudo_files.c:715", # fs/incfs/pseudo_files.c:715:10: warning: cast to pointer from integer of different size [-Wint-to-pointer-cast]
     "km_apphint.c:230", # drivers/staging/imgtec/rogue/km_apphint.c:230:48: warning: division 'sizeof (void *) / sizeof (void)' does not compute the number of array elements [-Wsizeof-pointer-div]
     "file.c:3010", # fs/f2fs/file.c:3010:12: warning: 'f2fs_ioctl_check_project' defined but not used
-- 
2.35.3

